pyinfra/operations/files.py

```diff
@@ -778,15 +778,37 @@ def template(
             group='root',
         )
 
-        # Example showing how to pass python variable to template file.
+        # Example showing how to pass python variable to template file. You can also use dicts and lists
         # The .j2 file can use `{{ foo_variable }}` to be interpolated.
```

Comment:
```suggestion
        # Example showing how to pass python variable to template file. You can also
        # use dicts and lists. The .j2 file can use `{{ foo_variable }}` to be interpolated.
```

This should fix the linting error (line too long).